packages/postcss-svgo/src/__tests__/index.js

```diff
@@ -152,6 +152,13 @@ test(
     'h1{background-image:url("data:image/svg;charset=US-ASCII,<?xml version="1.0" encoding="utf-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve"><circle cx="50" cy="50" r="40" fill="yellow" /><!--test comment--></svg>")}',
 );
 
+test('should warn on SVG containing unclosed tags', async (t) => {
+    const css =
+      'h1{background:url(data:image/svg+xml;charset=utf-8,<svg>style type="text/css"><![CDATA[ svg { fill: red; } ]]></style></svg>)}';
+    const result = await postcss(plugin()).process(css, {from: undefined});
+    t.deepEqual(result.messages.length, 1);
```

Comment:
Good that this test is here, although `deepEqual()` is strange (there's nothing 'deep' about `1`). But it does not hurt wither I guess.

Reply:
fixed to `t.is`